ocaml/idl/datamodel_vtpm.ml

```diff
@@ -79,5 +101,6 @@ let t =
       ; field ~qualifier:DynamicRO ~ty:(Ref _secret) ~internal_only:true
           ~lifecycle:[] "contents" "The contents of the TPM"
       ]
-    ~messages:[create; destroy; get_contents; set_contents]
+    ~messages:
+      [create; destroy; get_contents; set_contents; assert_operation_valid]
```

Comment:
I think there is a shortcut to declare allowed operations: `allowed_and_current_operations operations` that should generate some of the boilerplate.

Reply:
Yes, that's a helper function to add the two fields.